Backpack-SwiftUI/FlightLeg/Classes/BPKFlightLeg.swift

```diff
@@ -0,0 +1,177 @@
+import SwiftUI
+import Backpack_Common
+
+struct BPKFlightLeg: View {
+    private let airlineLogoSize: CGFloat = 24.0
+
+    private let departureArrivalTime: String
+    private let nextDayArrival: String?
+    private let flightDescription: AttributedString
+    private let stopsInfo: String
+    private let highlightStopsInfo: Bool
+    private let duration: String
+    private let operatedBy: String?
+    private let warning: String?
+    @State private var carrierLogo: Image?
```

Comment:
does this need to be `@State`? seems it does not, a regular `let` should suffice?
also, you should store the UIImage instead of an Image and create the Image(uiImage: image) on the view's body instead I think.